wicker/core/column_files.py

```diff
@@ -199,34 +237,25 @@ def __init__(
         :param dataset_name: name of the dataset, defaults to None
         :type dataset_name: str, optional
         """
```

Comment:
@pickles-bread-and-butter nit to update the docstring

Reply:
Done, thanks for pointing it out.